docs/General/install.rst

```diff
@@ -8,8 +8,23 @@ Freeciv21 has a number of prerequisites.  Note, that apart from the first prereq
 configuration process is smart enough to work out whether your system is suitable. If in doubt, just try it.
 
 An operating system that support Qt
-    Any modern operating system that support Qt is required. As of this writing this is Linux, Microsoft
-    Windows\ |reg| and Apple Mac OS X\ |reg|. On Windows Msys2 is supported.
+    Any modern operating system that supports Qt 5.12+ is required. As of this writing this is Linux, Microsoft
+    Windows\ |reg| and Apple Mac OS X\ |reg|. On Windows MSYS2 (MingW) is supported.
+
+    Linux Distributions:
+
+    * Arch Current
+    * CentOS 8+
+    * Debian 11+ (Bullseye)
+    * Fedora 28+
+    * Gentoo
+    * KDE neon LTS

```

Comment:
Neon (capital N): the available editions are User, Testing, Unstable, and Developer. We support User but I guess the unstable ones will also work. Let's just say "KDE Neon"?

Reply:
ok will be fixed in next commit. I also got rid of "current" and "stable"